.vscode/launch.json

```diff
@@ -199,17 +199,15 @@
             "console": "internalConsole"
         },
         {
-            "name": "server: RQ - scheduler",
+            "name": "spectr",
             "type": "python",
             "request": "launch",
             "stopOnEntry": false,
             "justMyCode": false,
             "python": "${command:python.interpreterPath}",
             "program": "${workspaceRoot}/manage.py",
             "args": [
-                "rqscheduler",
-                "--queue",
-                "export"
+                "spectacular",
```

Comment:
What's this for? There already is a launcher configuration that runs `spectacular`.